plugins/VanillaStats/class.vanillastats.plugin.php

```diff
@@ -151,6 +154,55 @@ public function settingsController_home_create($sender) {
      * period. This gets ajaxed into the dashboard homepage as date ranges are defined.
      */
     public function settingsController_dashboardSummaries_create($Sender) {
+        $DiscussionData = [];
+        $UserData = [];
+        $isVanillaAnalyticEnabled = Gdn::addonManager()->isEnabled('vanillaanalytics', Vanilla\Addon::TYPE_ADDON);
+
+        if (c('Garden.Analytics.DashboardSummaries', !$isVanillaAnalyticEnabled)) {
```

Comment:
This config is checked and its default prepared in almost the same way in two different places. How about moving this logic into the plugin constructor and saving the value in a new property on the class, similar to how the `AnalyticsServer` and `VanillaID` properties are handled?

```php
public function __construct() {
    ...
    $vanillaAnalyticEnabled = Gdn::addonManager()->isEnabled('vanillaanalytics', Vanilla\Addon::TYPE_ADDON);
    $this->doDashboardSummaries = c('Garden.Analytics.DashboardSummaries', !$isVanillaAnalyticEnabled);
}
```